lib/src/connector/connector.cc

```diff
@@ -305,6 +305,13 @@ bool Connector::isAssociated() const
     return isConnected() && session_association_.success.load();
 }
 
+ConnectionTimings Connector::getConnectionTimings() const
+{
+    return (connection_ptr_ == nullptr
+                ? ConnectionTimings()
```

Comment:
I wonder where this ConnectionTimings() object is created - on the stack or heap?